# ====== DO NOT MODIFY! AUTOGENERATED FROM AN ACTION DEFINITION ======

ClawOrderActionGoal action_goal
ClawOrderActionResult action_result
ClawOrderActionFeedback action_feedback
